platform/smart-home-app.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Stub for App class
+ */
+/// <reference types="@google/local-home-sdk" />
+
+//TODO(cjdaly) fill in implementation of these methods
+export const AppStub = class {
```

Comment:
should this implement the App interface from the platform?

Reply:
Yes,  Added implementation, but going to fill in methods in another PR (#24)